Makefile

```diff
@@ -4,17 +4,17 @@
 DOCNAME = UCDlist
 
 # count up; you probably do not want to bother with versions <1.0
-DOCVERSION = 1.4
+DOCVERSION = 1.5
 
 # Publication date, ISO format; update manually for "releases"
-DOCDATE = 2021-06-16
+DOCDATE = 2022-05-03
 
 # What is it you're writing: NOTE, WD, PR, REC, PEN, or EN
-DOCTYPE = EN
+DOCTYPE = PEN
 
 # An e-mail address of the person doing the submission to the document
 # repository (can be empty until a make upload is being made)
-AUTHOR_EMAIL=baptiste.cecconi@obspm.fr
+AUTHOR_EMAIL=baptiste.cecconi@obspm.fr, mireille.louys@unistra.fr
```

Comment:
This is used by the IVOA document repository and needs to be a single mail address -- please just put in yourself only here.